hsc/crawler.py

```diff
@@ -80,13 +80,19 @@ def login(self, username, password):
 		resp = self.session.get(self.login_url, auth=(username, password))
 		self.cookies = self.session.cookies.get_dict()
 		self.headers = resp.request.headers
-		return resp.url != self.login_url
+		self.get_number_of_submissions()
+		return self.total_number_of_submissions != 0
 
 	def authenticate(self):
 		username = input('Hackerrank Username: ')
 		password = getpass.getpass('Hackerrank Password: ')
 		return self.login(username, password)
 
+	def get_number_of_submissions(self):
```

Comment:
Usually getters are supposed to return value. You can change this function to set total number of submissions if not found and then return total number of submissions.

Reply:
This is not a getter. I've used `get` to mean `fetch`. Should I change the method name?

Reply:
renaming to fetch still doesn't signify setting value. Fetch also usually means a get operation over a network.